### ISSUE TITLE
HOL Step-by-step

### ISSUE THREAD

**Issue by DawnmarieDesJardins** [Microsoft/MCW-Media-AI#14]
Line 282 - Image does not have alt text
Line 302 - no alt-text
Line 1214 - please check coding, something is off but I don't understand the instructions well enough to correct it.

**Comment by crpietschmann** [comment]
fixes have been made
